src/Microsoft.AspNetCore.Mvc.Core/Internal/ObjectMethodExecutor.cs

```diff
@@ -31,6 +33,48 @@ public class ObjectMethodExecutor
 
         private static readonly MethodInfo _coerceMethod = ((MethodCallExpression)_coerceTaskExpression.Body).Method;
 
+        private object[] ParameterDefaultValues
```

Comment:
No need for the additional property. Instead create a `private void EnsureParameterDefaultValues()` method at the bottom of the file and call it before using `_parameterDefaultValues`.

Note that if the `private` property were necessary, it should be added just below the existing properties.